### DISCUSSION TITLE
hotspot UI in iframe?

### DISCUSSION THREAD

**Discussion by outdoorbits** [davesteele/comitup#256] [General]
I would like to run comitup hotspot UI in an iframe. This would allow to use comitup in little-backup-box having a touch screen. I just tried this:
`<iframe id="pageframe" src="http://192.168.178.86"></iframe>`
The iframe is empty but the link http://192.168.178.86 works.

Thank you!

**Comment by davesteele** [comment]
I've not tried anything like this. I sounds like it should work.
